brands/amenity/bureau_de_change.json

```diff
@@ -38,6 +38,17 @@
       "name": "No1 Currency"
     }
   },
+  "amenity/bureau_de_change|PocketChange": {
+    "locationSet": {"include": ["jp", "tw"]},
+    "tags": {
+      "amenity": "bureau_de_change",
```

Comment:
Does it make sense to tag coin to bill converters as bureau_de_change? Is there potentially a better way to tag these since there isn't actual currency conversion happening here as described in the wiki.

Reply:
It is honestly a machine that performs currency conversions (dollar, yen and euro). It is not an ATM because it is not anchored to a bank. Nor is it Western Union. So it's the label that comes closest.